src/app/shared/log-in/log-in.component.scss

```diff
@@ -0,0 +1,15 @@
+@import '../../../styles/variables.scss';
+
+.form-login .form-control:focus {
+  z-index: 2;
+}
+.form-login input[type="email"] {
+  margin-bottom: -1px;
+  border-bottom-right-radius: 0;
```

Comment:
Can be achieved using [the `rounded-bottom` bootstrap class](https://getbootstrap.com/docs/4.0/utilities/borders/#border-radius)